.gitignore

```diff
@@ -89,6 +89,7 @@ packages/core/src/Site/*.js
 packages/core/src/utils/*.js
 packages/core/src/variables/*.js
 packages/core/test/unit/**/*.js
+packages/core/_site/*
```

Comment:
This should not be necessary, why are we creating a markbind site in core?

Reply:
This happens because some test cases fail to generate the diagram before the program hits a fatal error, leaving the lockfile unremoved. As a result, there are some dummy files inside the `packages/core/_site/*`. If we apply the Singleton implementation, this would be redundant.

Reply:
So is it still redundant or do we need to keep this?